x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/store/input/FrozenIndexInput.java

```diff
@@ -166,7 +166,8 @@ private void readWithoutBlobCacheSlow(ByteBuffer b, long position, int length) t
                         relativePos,
                         len,
                         progressUpdater,
-                        writeBuffer.get().clear()
+                        writeBuffer.get().clear(),
+                        SharedBytes.CacheCopyMetricsConsumer.NO_OP
```

Comment:
I got the impression we're not interested in supporting this for frozen tier as part of this change, but I believe it would be trivial to implement.